### ISSUE TITLE
shell_fallback setting is being put in incorrect section in sssd.conf

### ISSUE THREAD

**Issue by rbogdany** [chef-cookbooks/sssd_ldap#31]
### Cookbook version
4.0.0

### Chef-client version
12.9.38

### Platform Details
RHEL7.3

### Scenario:
sssd is producing this warning in /var/log/sssd/sssd.log:
```
(Tue Jun 27 20:48:48:733012 2017) [sssd] [sss_ini_call_validators] (0x0020): [rule/allowed_domain_options]: Attribute 'shell_fallback' is not allowed in section 'domain/LDAP'. Check for typos.
```

### Steps to Reproduce:
Just a simple `include_recipe 'sssd_ldap'` will produce an /etc/sssd/sssd.conf with the shell_fallback attribute in the [domain/LDAP] section.

### Expected Result:
According to the RHEL7 man page on sssd.conf, the shell_fallback option is only allowed in the nss section.

### Actual Result:
sssd is producing this warning in /var/log/sssd/sssd.log:
```
(Tue Jun 27 20:48:48:733012 2017) [sssd] [sss_ini_call_validators] (0x0020): [rule/allowed_domain_options]: Attribute 'shell_fallback' is not allowed in section 'domain/LDAP'. Check for typos.
```

**Comment by tas50** [comment]
I’m adding the Type: Jump In GitHub label to this issue. This is a great issue for someone to get their feet wet with and we’d love a PR to resolves the issue.

**Comment by tas50** [comment]
This has been fixed in the 4.2.1 release by removing that attribute since it never actually worked. There's another open PR that needs to be rebased to configure the NSS block where this config belongs
